mava/utils/make_env.py

```diff
@@ -32,20 +36,27 @@
 from jumanji.environments.routing.robot_warehouse.generator import (
     RandomGenerator as RwareRandomGenerator,
 )
+from lbforaging.foraging import environment as gym_lbf
```

Comment:
Does LBF work for you? I tested with the branch used for the ff_ippo PR and got this error `TypeError: 'module' object is not callable`. This should fix it:
```suggestion
from lbforaging.foraging import ForagingEnv as gym_lbf
```

And please fix the arguments naming in all configs, ie `num_agents -> players ` 
You'll find more info on the right naming [here](https://github.com/LukasSchaefer/lb-foraging/blob/master/lbforaging/__init__.py 🙌

Reply:
I forgot to re-test lbf after the config file changes, everything should be working as intended now!